com.is_slick.rmac.rmac.yml

```diff
@@ -0,0 +1,28 @@
+id: com.is_slick.rmac.rmac
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: rmac
+modules:
```

Comment:
Does this require no permissions? If so you need to open an exception see https://docs.flathub.org/docs/for-app-authors/linter#exceptions for https://docs.flathub.org/docs/for-app-authors/linter#finish-args-not-defined

Reply:
It needs to access files on the filesystem for reading and writing. I thought in this case I didn't need to do anything. I'll consult the links you provided, thanks.

Reply:
> t needs to access files on the filesystem for reading and writing. 

Then you need add that https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

Reply:
> It needs to access files on the filesystem for reading and writing. I thought in this case I didn't need to do anything. I'll consult the links you provided, thanks.

So you didn't try it. You justify not building from source that you have to run a test suite, but then don't test the resulting package anyway.